### PR TITLE
Blog ksc nominations

### PR DESCRIPTION
**Pull request by csantanapr**
Since blog posts go out thru RSS, I wanted to help publicize more the Knative Steering Committee elections.
And with the deadline for nominations and not that many candidates currently I think this might help.

Anyone see a problem publishing this today?

### CODE REVIEW

File: blog/steering/2020-11-ksc-elections.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+| November 9   | All candidate nominations due by 0000 UTC (5pm Pacific) |
+| November 12  | Election Begins via email ballots |
+| November 22  | Last call for exception & replacement ballots |
+| November 29  | Election Closes by 0000 UTC (5pm Pacific) |
+| December 3   | Announcement of Results at Knative Steering Meeting |
+
+
+You can read the current [Knative Steering Committee Election - 2020 Voters Guide](https://github.com/knative/community/blob/master/elections/2020/SC/README.md) for all the details.
+
+And remember if you have the right to vote, speak your mind and vote because every vote counts!
```
Comment (rhuss) [inline blog/steering/2020-11-ksc-elections.md:51]: Sorry to comment on the linting check below, but I really wonder who was ever hurt by a trailing newline of a document so much that it justifies an extra turn around ? It's not nice, sure, but was is the reasoning that a missing newline at the end of a document needs to break the build ? Which modern editor has an issue with those or even expose this as an error ? Really curious, as I have seen that now several times and IMO it's just a waste to time to fix this (or write these kind of comments). tbh, most of the "markdown checks" are much more harmful than helpful. It's prose.

Comment (evankanderson) [reply blog/steering/2020-11-ksc-elections.md:51]: I'd put a lot of lint checks in that boat -- particularly ones that could either be automatic fixes or just ignored.

**Review by evankanderson — COMMENTED** [review]
/lgtm

I'll let @macruzbar approve this.

**Comment by evankanderson** [comment]
/lgtm
/approve

So that we can get a weekend day off this being up before nominations close.

**Pull request merged**